diff --git a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
index 97ed962..a035a24 100644
--- a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
+++ b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
@@ -412,7 +412,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -460,7 +460,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -534,7 +534,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
@@ -577,7 +577,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
