diff --git a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
index 6dd53bf..fea7ef4 100644
--- a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
+++ b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
@@ -242,9 +242,11 @@ custom_edit_url: null
                             .toString("base64"));
                     let infoBasePath = `${outputDir}/${item.infoId}`;
                     if (docRouteBasePath) {
-                        infoBasePath = `${docRouteBasePath}/${outputDir
-                            .split(docPath)[1]
-                            .replace(/^\/+/g, "")}/${item.infoId}`.replace(/^\/+/g, "");
+                        infoBasePath = outputDir.indexOf(docPath) > -1
+                            ? `${docRouteBasePath}/${outputDir
+                                .split(docPath)[1]
+                                .replace(/^\/+/g, "")}/${item.infoId}`.replace(/^\/+/g, "")
+                            : infoBasePath;
                     }
                     if (item.infoId)
                         item.infoPath = infoBasePath;
@@ -414,7 +416,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -462,7 +464,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -537,7 +539,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
@@ -580,7 +582,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
