diff --git a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
index 90a4d82..2d125dd 100644
--- a/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
+++ b/node_modules/docusaurus-plugin-openapi-docs/lib/index.js
@@ -244,9 +244,11 @@ custom_edit_url: null
                             .toString("base64"));
                     let infoBasePath = `${outputDir}/${item.infoId}`;
                     if (docRouteBasePath) {
-                        infoBasePath = `${docRouteBasePath}/${outputDir
-                            .split(docPath)[1]
-                            .replace(/^\/+/g, "")}/${item.infoId}`.replace(/^\/+/g, "");
+                        infoBasePath = outputDir.indexOf(docPath) > -1
+                            ? `${docRouteBasePath}/${outputDir
+                                .split(docPath)[1]
+                                .replace(/^\/+/g, "")}/${item.infoId}`.replace(/^\/+/g, "")
+                            : infoBasePath;
                     }
                     if (item.infoId)
                         item.infoPath = infoBasePath;
@@ -473,7 +475,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -527,7 +529,7 @@ custom_edit_url: null
                 let targetDocsPluginId;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                         targetDocsPluginId = pluginConfig.docsPluginId;
                     }
@@ -604,7 +606,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
@@ -653,7 +655,7 @@ custom_edit_url: null
                 let targetConfig;
                 if (pluginId) {
                     try {
-                        const pluginConfig = getPluginConfig(plugins, pluginId);
+                        const pluginConfig = getPluginConfig(pluginInstances, pluginId);
                         targetConfig = (_a = pluginConfig.config) !== null && _a !== void 0 ? _a : {};
                     }
                     catch {
